sails/api/hooks/custom/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * custom hook
+ *
+ * @description :: A hook definition.  Extends Sails by adding shadow routes, implicit actions, and/or initialization logic.
```

Comment:
Copy/paste at its best, maybe short title in line 2 and here a good description.